src/main/java/org/broadinstitute/consent/http/resources/NihAccountResource.java

```diff
@@ -49,11 +45,11 @@ public Response syncAccount(@Auth DuosUser duosUser) {
   @Produces(MediaType.APPLICATION_JSON)
   @Consumes(MediaType.APPLICATION_JSON)
   @PermitAll
-  public Response registerResearcher(@Auth AuthUser authUser, NIHUserAccount nihAccount) {
+  public Response registerResearcher(@Auth DuosUser duosUser, NIHUserAccount nihAccount) {
     try {
-      nihService.validateNihUserAccount(nihAccount, authUser);
-      User user = userService.findUserByEmail(authUser.getEmail());
-      List<UserProperty> authUserProps = nihService.authenticateNih(nihAccount, authUser,
+      nihService.validateNihUserAccount(nihAccount, duosUser);
+      User user = duosUser.getUser();
+      List<UserProperty> authUserProps = nihService.authenticateNih(nihAccount, duosUser,
```

Comment:
Florian's PR removes this altogether, there will need to be some syncing up soon when we start merging RAS code.